crates/router/src/connector/riskified/transformers/api.rs

```diff
@@ -156,14 +186,19 @@ impl TryFrom<&frm_types::FrmCheckoutRouterData> for RiskifiedPaymentsCheckoutReq
                 created_at: common_utils::date_time::now(),
                 updated_at: common_utils::date_time::now(),
                 gateway: payment_data.request.gateway.clone(),
-                total_price: payment_data.request.amount,
+                total_price: payment.amount.clone(),
                 cart_token: payment_data.attempt_id.clone(),
                 line_items: payment_data
                     .request
                     .get_order_details()?
                     .iter()
                     .map(|order_detail| LineItem {
-                        price: order_detail.amount.get_amount_as_i64(), // This should be changed to MinorUnit when we implement amount conversion for this connector. Additionally, the function get_amount_as_i64() should be avoided in the future.
+                        price: convert_amount(
+                            payment.amount_converter,
+                            MinorUnit::new(order_detail.amount),
+                            common_enums::Currency::USD,
+                        )
+                        .unwrap(),
```

Comment:
unwrap should not be used they can cause programs to halt unexpectedly. Instead you can propagate the error.